### PR TITLE
[10.0] Added analytic tags to contracts

### PR DESCRIPTION
**Pull request by katyukha**
This commit adds ability to specify analytic tags on contracts invoice
lines, which will be propagated to invoice lines.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rebase and solve conflicts.

**Comment by katyukha** [comment]
Rebased

File: contract/models/account_analytic_account.py [hunk 1]
```diff
@@ -188,9 +188,14 @@ def _prepare_invoice_line(self, line, invoice_id):
             'uom_id': line.uom_id.id,
             'discount': line.discount,
         })
+
```
Comment (pedrobaeza) [inline contract/models/account_analytic_account.py]: Don't add line breaks inside methods

File: contract/models/account_analytic_account.py [hunk 2]
```diff
@@ -188,9 +188,14 @@ def _prepare_invoice_line(self, line, invoice_id):
...
+        # Add analytic tags to invoice line
+        invoice_line.analytic_tag_ids |= line.analytic_tag_ids
+
         # Get other invoice line values from product onchange
         invoice_line._onchange_product_id()
         invoice_line_vals = invoice_line._convert_to_write(invoice_line._cache)
+
```
Comment (pedrobaeza) [inline contract/models/account_analytic_account.py]: Remove

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please change module version number according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

**Comment by katyukha** [comment]
@pedrobaeza For module version what part of number is better to increase?
10.0.x.y.z : *x* or *y*? There only one field added, and module upgrade is required, but no migration needed.

Thanks for your review

**Review by carlosdauden — APPROVED** [review]
Remove blank lines and change version

**Comment by katyukha** [comment]
There is strange error on runbot

**Comment by pedrobaeza** [comment]
Yeah, strange, and Travis has not been fired. Please change module version raising `y` letter, and see if now it works correctly.

**Comment by katyukha** [comment]
Tests passed

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**